hydrocarbon_problem/agents/sac/README.md

```diff
@@ -0,0 +1,31 @@
+# SAC Agent
```

Comment:
Here is a high level description of things, please leave comments for any parts of the code that are unclear.